apps/psp-front/tailwind.config.js

```diff
@@ -3,7 +3,84 @@ module.exports = {
   plugins: [],
   purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
   theme: {
-    extend: {},
+    colors: {
+      transparent: 'transparent',
+      white: '#ffffff',
+      whitesmoke:'#f5f5f5',
```

Comment:
please check if this removes other tailwind colors :)
if it does, move all colors from theme to theme:{extend:{......}} part